parse_node/parse_import_declaration.ts

```diff
@@ -46,6 +46,12 @@ export const getImportResPathForEnum = (
 
   const enumDeclarations = enumSymbol.declarations
 
+  if (!enumDeclarations) {
+    throw new Error(
+      `No Enum declartion given`
```

Comment:
Hm, were you running into this issue in practice? Or is this just to satisfy the typechecker?

Reply:
TS changed the declartions to being optional, so just satisfying the checker here.